From 4f3f9d516c68c99b412b6e4b2eaa6e25f389b337 Mon Sep 17 00:00:00 2001
From: Adam Majer <amajer@suse.de>
Date: Tue, 22 May 2018 09:43:21 +0200
Subject: [PATCH] Add missing pthread library for server

While building snapper with Boost 1.67, the following error
occurred,

/usr/lib64/gcc/x86_64-suse-linux/7/../../../../x86_64-suse-linux/bin/ld: Client.o: undefined reference to symbol 'pthread_condattr_setclock@@GLIBC_2.3.3'
/lib64/libpthread.so.0: error adding symbols: DSO missing from command line
collect2: error: ld returned 1 exit status
make[2]: *** [Makefile:440: snapperd] Error 1
make[2]: Leaving directory '/home/abuild/rpmbuild/BUILD/snapper-0.5.4/server'
---
 server/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/server/Makefile.am b/server/Makefile.am
index 2a6a4c2d..009fe414 100644
--- a/server/Makefile.am
+++ b/server/Makefile.am
@@ -14,4 +14,4 @@ snapperd_SOURCES =					\
 	Types.cc		Types.h
 
 snapperd_LDADD = ../snapper/libsnapper.la ../dbus/libdbus.la -lrt
-snapperd_LDFLAGS = -lboost_system -lboost_thread
+snapperd_LDFLAGS = -lboost_system -lboost_thread -lpthread
